practice/validator/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Validator</title>
+  </head>
+  <body>
+        <form>
+          <p>
+            Name: First Last
+            <input class="fullname" type="text">
+          </p>
+          <p>
+            Date of birth: YYYY-MM-DD
+            <input class="birthday" type="text">
+          </p>
+          <p>
+            Phone Number: ###-###-####
+            <input class="phone-number" type="text">
+          </p>
```

Comment:
Just gonna pass on some advice Andrew posted to mine:
'Remember to use the <label> tag, which makes input fields bigger/easier targets for finger taps and mouse clicks.

It's both a convenience and an accommodation for people with various disabilities that prevent them from doing it the "normal" way.'